guides/coco-dev.md

```diff
@@ -77,16 +77,16 @@ and confidential hardware.
 
 CoCo has a modular attestation interface and there are a few options for attestation.
 CoCo provides a generic Key Broker Service (KBS) that the rest of this guide will be focused on.
-The SEV runtime class uses `simple-kbs`, which is described in the [SEV guide](../guides/sev.md).
+The SEV runtime class uses `simple-kbs`, which is described in the [SEV guide](https://confidentialcontainers.org/docs/examples/snp-container-launch/).
```

Comment:
This link doesn't seem correct - it points to the SNP page on the website that isn't relevant to simple-kbs?